flashinfer/quantization/fp4_quantization.py

```
@@ -747,6 +747,50 @@ def fp4_quantize(
     return x_q, sf
 
 
+@torch.library.custom_op("flashinfer::fp4_quantize", mutates_args=())
+def _fp4_quantize_custom_op(
+    input: torch.Tensor,
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)